lizard_auth_server/models.py

```diff
@@ -7,7 +7,14 @@
 from django.db import models
 from django.db import transaction
 from django.db.models import F
-from django.db.models.loading import get_model
+
+try:
+    # Django >=1.9
```

Comment:
Wat mij betreft mag je in de `setup.py` gewoon vereisen `django >= 1.9`, dan scheelt dat hier een if/else.